src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java

```diff
@@ -1,30 +1,31 @@
 package ru.skypro.homework.service.impl;
 
 import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.security.provisioning.UserDetailsManager;
 import org.springframework.stereotype.Service;
 import ru.skypro.homework.service.AuthService;
 
 @Service
 public class AuthServiceImpl implements AuthService {
 
-    private final UserDetailsManager manager;
+    private final UserDetailsService userDetailsService;
     private final PasswordEncoder encoder;
 
-    public AuthServiceImpl(UserDetailsManager manager,
+    public AuthServiceImpl(UserDetailsServiceImpl userDetailsService,
```

Comment:
```suggestion
    public AuthServiceImpl(UserDetailsService userDetailsService,
```
инъекцию надо делать по интерфейсу